src/nvim/eval.c

```diff
@@ -17142,10 +17144,18 @@ static void f_timer_stopall(typval_T *argvars, typval_T *unused, FunPtr fptr)
 static void timer_due_cb(TimeWatcher *tw, void *data)
 {
   timer_T *timer = (timer_T *)data;
+  int save_did_emsg = did_emsg;
+  int save_called_emsg = called_emsg;
+
   if (timer->stopped || timer->paused) {
     return;
   }
 
+  // Don't run timer while dealing with an error
+  if (aborting()) {
```

Comment:
This is not needed (doesn't correspond to what the `aborting()` check in the vim patch does).